system/system_monitor/src/net_monitor/net_monitor.cpp

```diff
@@ -81,7 +82,11 @@ NetMonitor::NetMonitor(const rclcpp::NodeOptions & options)
   get_reassembles_failed_column_index();
 
   // Send request to start nethogs
-  send_start_nethogs_request();
+  if (enable_traffic_monitor_) {
+    send_start_nethogs_request();
+  } else {
+    send_stop_nethogs_request();
```

Comment:
The current function name feels off. I think send_skip_nethogs_request would be better.

Reply:
Fixed in fad266d995c46e12f8ff763e4db67cb19a64298d